docs/backends.md

```diff
@@ -72,6 +78,11 @@ Content-Length: 165
 ```
 
 ## View Backend
+An individual backend can be viewed on ui where it's details such as address, status, ID, certs and information can be acquired. Backend termination is also done on this page through cert revocation.
```

Comment:
cert revocation is not always equal to termination, also just mention that this page includes cert management